sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/EncryptResult.java

```diff
@@ -0,0 +1,55 @@
+package com.azure.security.keyvault.keys.cryptography;
```

Comment:
Licensing header, check on this in all files added

Reply:
Fixed,
But checkstyle fixes coming in the batch at the end today, assume they don't exist.